app/domain/authentication/authenticate.rb

```diff
@@ -85,19 +84,6 @@ def audit_success
       )
     end
 
-    def audit_failure(err)
-      @audit_log.log(
-        ::Audit::Event::Authn::Authenticate.new(
-          authenticator_name: authenticator_name,
-          service: webservice,
-          role_id: audit_role_id,
-          client_ip: client_ip,
-          success: false,
-          error_message: err.message
-        )
-      )
-    end
-
     def audit_role_id
```

Comment:
do we still use this? please verify we have no audit leftovers in the files you change